projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.html

```diff
@@ -27,38 +25,45 @@
   <div>
     <cdk-tree [dataSource]="nestedDataSource" [treeControl]="nestedTreeControl">
       <!-- document -->
-      <cdk-nested-tree-node class="document-tree-container__node document-tree-container__node--document" *cdkTreeNodeDef="let node">
+      <cdk-nested-tree-node class="
+          document-tree-container__node document-tree-container__node--document
+        " *cdkTreeNodeDef="let node">
         <button class="node" (click)="selectedNodeItem = node; clickedDocument.emit(node)"
-                [class.node--selected]="selectedNodeItem?.name === node.name">
+          [class.node--selected]="selectedNodeItem?.name === node.name">
           <div class="node__icon" disabled>
-            <img src="/assets/img/case-file-view/case-file-view-document.svg" class="node__iconImg" alt="Document icon">
+            <img src="/assets/img/case-file-view/case-file-view-document.svg" class="node__iconImg"
+              alt="Document icon" />
           </div>
           <span class="node__name node-name-document">
-            {{node.name}}
-            <br>
-            <span class="node__document-upload-timestamp">{{node.upload_timestamp | ccdDate : 'local' :'dd MMM YYYY HH:mm'}}</span>
+            {{ node.name }}
+            <br />
+            <span class="node__document-upload-timestamp">{{
+              node.upload_timestamp | ccdDate: "local":"dd MMM YYYY HH:mm"
+              }}</span>
           </span>
           <div class="node__document-options">
             <ccd-case-file-view-folder-document-actions
-              (changeFolderAction)="triggerDocumentAction('changeFolder', node)"
-              (openInANewTabAction)="triggerDocumentAction('openInANewTab', node)"
-              (downloadAction)="triggerDocumentAction('download', node)"
-              (printAction)="triggerDocumentAction('print', node)"
-              [allowMoving]="allowMoving"
-            >
+              (changeFolderAction)="triggerDocumentAction('changeFolder', node)" (openInANewTabAction)="
```

Comment:
weird formatting change, and not an improvement. Can you put it back the way it was, with each param on a separate line